### PR TITLE
Sexy Clown

### PR DESCRIPTION
**Pull request by Emisse**
changelog
-sexy clown outfit

### CODE REVIEW

**Comment by ikamuse** [comment]
uh oh

**Comment by Elijahrane** [comment]
░░░░░░░░░░░░▄▄                     
░░░░░░░░░░░█░░█                  
░░░░░░░░░░░█░░█                  
░░░░░░░░░░█░░░█                  
░░░░░░░░░█░░░░█                  
███████▄▄█░░░░░██████▄ 
▓▓▓▓▓▓█░░░░░░░░░░░░░░█ 
▓▓▓▓▓▓█░░░░ Valve, ░░░░░█ 
 ▓▓▓▓▓▓█░░░░Add This░░░░█  
▓▓▓▓▓▓█░░░░Please!░░░░░█ 
▓▓▓▓▓▓█░░░░░░░░░░░░░░█ 
▓▓▓▓▓▓█████░░░░░░░░░█   
██████▀░░░░▀▀██████▀

**Comment by LordEclipse** [comment]
Big need.

**Comment by metalgearsloth** [comment]
you really had to make this pr#10000

**Comment by Ablankmann** [comment]
society when the next PR would be PR #10000

File: Resources/Prototypes/Catalog/VendingMachines/Inventories/theater.yml [hunk 1]
```diff
@@ -5,6 +5,7 @@
   startingInventory:
     ClothingOuterWinterClown: 1
     ClothingOuterWinterMime: 1
+    ClothingMaskSexyClown: 1
```
Comment (Elijahrane) [inline Resources/Prototypes/Catalog/VendingMachines/Inventories/theater.yml:8]: You forgot the jumpskirt too

**Review by Elijahrane — CHANGES_REQUESTED** [review]

**Comment by keronshb** [comment]
Fetish content smh....

**Comment by moonheart08** [comment]
oh my god

**Comment by optimumtact** [comment]
needs in game pictures

**Comment by ThunderBear2006** [comment]
> Fetish content smh....

Hey man don't judge the clussy lovers, everyone has their kink lmao

**Comment by Kromabae** [comment]
ngl if you just removed the word sexy everywhere, this might acutally be a fine edition

File: Resources/Prototypes/Catalog/VendingMachines/Inventories/theater.yml (hunk 1 above)
Comment (Emisse) [reply Resources/Prototypes/Catalog/VendingMachines/Inventories/theater.yml:8]: jumpskirt spawns when you select jumpskirt option

**Comment by metalgearsloth** [comment]
no fetish content

**Comment by Emisse** [comment]
its not

**Comment by Just-a-Unity-Dev** [comment]
i can confirm this isnt fetish contnet

**Comment by Emisse** [comment]
ok its fetish content im pushing fetish content on upstream!!!!!! MERG ENOW

**Comment by 5150Con** [comment]
Is appreciating a fine clown a fetish? Is enjoying the sunrise a fetish? The clown transcends mere eroticism and borders more on the ethereal, and while those of low knowledge may equate this to some mere physical attraction, the PR is created for those who understand the ephemeral beauty of the Honkmother and her creations. Merging this PR would surely elevate the lives of those on Wizard's Den, and mark the community as one that stand for art and pursing the human purpose.

**Review by Veritius — APPROVED** [review]

**Review by Elijahrane — APPROVED** [review]

**Comment by mirrorcult** [comment]
PR author has been executed for crimes against humanity so no merging is required

**Comment by EmoGarbage404** [comment]
Maintainer found mysteriously dead in home. Signs of forced entry.

**Pull request merged**